assets/scripts/works.js

```diff
@@ -0,0 +1,284 @@
+    wbutton2.appendChild(wbutton2Text);
+    const wimg5 = document.createElement('img');
+    wimg5.src = './assets/images/live_icon.svg';
+    wbutton2.appendChild(wimg5);
+    wform2.appendChild(wbutton2);
+    wdiv4.appendChild(wform2);
+    const wform3 = document.createElement('form');
+    wform3.action = projectSource;
+    const wbutton3 = document.createElement('button');
+    wbutton3.className = 'detail-buttons-desktop source';
+    const wbutton3Text = document.createTextNode('See Source');
+    wbutton3.appendChild(wbutton3Text);
+    const wimg6 = document.createElement('img');
+    wimg6.src = './assets/images/source_icon.svg';
+    wbutton3.appendChild(wimg6);
+    wform3.appendChild(wbutton3);
+    wdiv4.appendChild(wform3);
+    wdiv3.appendChild(wdiv4);
+    wdiv2.appendChild(wdiv3);
+    const wul1 = document.createElement('ul');
+    wul1.className = 'toolkit';
+    const wli3 = document.createElement('li');
+    wli3.className = 'toolkit-list';
+    const wli3Text = document.createTextNode('Codekit');
+    wli3.appendChild(wli3Text);
+    wul1.appendChild(wli3);
+    const wli4 = document.createElement('li');
+    wli4.className = 'toolkit-list';
+    const wli4Text = document.createTextNode('GitHub');
+    wli4.appendChild(wli4Text);
+    wul1.appendChild(wli4);
+    const wli5 = document.createElement('li');
+    wli5.className = 'toolkit-list';
+    const wli5Text = document.createTextNode('JavaScript');
+    wli5.appendChild(wli5Text);
+    wul1.appendChild(wli5);
+    const wli6 = document.createElement('li');
+    wli6.className = 'toolkit-list';
+    const wli6Text = document.createTextNode('Bootstrap');
+    wli6.appendChild(wli6Text);
+    wul1.appendChild(wli6);
+    const wli7 = document.createElement('li');
+    wli7.className = 'toolkit-list';
+    const wli7Text = document.createTextNode('Terminal');
+    wli7.appendChild(wli7Text);
+    wul1.appendChild(wli7);
+    const wli8 = document.createElement('li');
+    wli8.className = 'toolkit-list';
+    const wli8Text = document.createTextNode('Codepen');
+    wli8.appendChild(wli8Text);
+    wul1.appendChild(wli8);
+    wdiv2.appendChild(wul1);
+    const wp1 = document.createElement('p');
+    wp1.className = 'detail-description';
+    const wp1Text = document.createTextNode(projectDescription);
+    wp1.appendChild(wp1Text);
+    wdiv2.appendChild(wp1);
+    wsection.appendChild(wdiv2);
+    const wbody = document.querySelector('body');
+    wbody.appendChild(wsection);
+    wbody.style.background = '#c1c7d0';
+    document.querySelector('.main-portfolio').style.display = 'none';
+
+    function hideWork() {
+      wbody.removeChild(wsection);
+      wbody.style.background = '#fff';
+      document.querySelector('.main-portfolio').style.display = 'block';
+    }
+
+    document.querySelector('.main-image-mobile').addEventListener('click', hideWork);
+    document.querySelector('#detail-desktop-close-button').addEventListener('click', hideWork);
+  });
+});
```

Comment:
- [OPTIONAL] I would like to suggest that you should try to practice modular programming as having all your code in a file would make the file too bulky. In this case, you can have you projects array as a file export and import it to your main js file. You can as well the same for any other set of related functions.

Reply:
Thanks for the suggestion. I never used file export in JavaScript, before. Time to learn about it.